go_modules/lib/dependabot/go_modules/replace_stubber.rb

```diff
@@ -37,7 +37,6 @@ def stub_paths(manifest, directory)
       def stub_replace_path?(path, directory)
         return true if absolute_path?(path)
         return false unless relative_replacement_path?(path)
-        return true if @repo_contents_path.nil?
```

Comment:
This was the main check which prompted this PR. It was causing tests that should fail to pass because we weren't using a real repo.